docs/REST API/rest-api.md

```diff
@@ -4123,15 +4165,17 @@ Get aggregate status counts of Apple disk encryption profiles applying to macOS
 }
 ```
 
-### Get macOS settings statistics
+### Get OS settings status
+
+> [Get macOS settings statistics](https://github.com/fleetdm/fleet/blob/ee02782eaf84c121256d73abc20b949d31bf2e57/docs/REST%20API/rest-api.md#get-macos-settings-statistics) (`GET /api/v1/fleet/mdm/apple/profiles/summary`) API endpoint is deprecated as of Fleet 4.40. It is maintained for backwards compatibility. Please use the API endpoint.
 
-Get aggregate status counts of all macOS settings (configuraiton profiles and disk encryption) enforced on hosts.
+Get aggregate status counts of all OS settings (configuration profiles and disk encryption) enforced on hosts.
 
-For Fleet Premium uses, the statistics can
+For Fleet Premium users, the counts can
 optionally be filtered by team id. If no team id is specified, team profiles are excluded from the
```

Comment:
```suggestion
optionally be filtered by `team_id`. If no `team_id` is specified, team profiles are excluded from the
```

I think we want to start using `team_id` or team ID right?

Reply:
That's right. Thanks for noticing.